server/pools/views.py

```diff
@@ -67,7 +67,11 @@ def post(self, request):
         try:
             body = request.data
             pool = Pool.objects.get(pool_id=body['pool_id'])
-            if pool.can_place_reservation(body['slot_count'], body['start_datetime'], body['end_datetime']):
+            format_data_time = '%Y-%m-%d %H:%M:%S'
```

Comment:
ValueError: time data '2018-05-20 12:00' does not match format '%Y-%m-%d %H:%M:%S'

Testy nie przechodzą, funkcja rzuca errorami. Usunąć ":%S", sprawdzić czy potem wszystko działa.